src/core/meshcop/dataset.hpp

```diff
@@ -615,10 +615,8 @@ class Dataset
     /**
      * This constructor initializes the object.
      *
-     * @param[in]  aType       The type of the dataset, active or pending.
-     *
      */
-    explicit Dataset(Type aType);
+    explicit Dataset(void);
```

Comment:
Did you intend to remove the following? I don't see it removed in this PR. https://github.com/openthread/openthread/blob/71e00eb20f14992ecbbecab8e0c79cf5aa76bb9d/src/core/meshcop/dataset.hpp#L926